### DISCUSSION TITLE
Importing a Jupyter notebook into Hugo with ox-hugo?

### DISCUSSION THREAD

**Discussion by amca01** [kaushalmodi/ox-hugo#604] [Q&A]
Not really a bug, just a question.  I have a Jupyter notebook which contains a mixture of markdown cells (which themselves include LaTeX), and python cells, along with output.  It would be a pain to have to rewrite this all in org-mode from scratch, and I'm not sure if there's a Jupyter to org converter.  (There's a body of thought that believes Jupyter should be dropped for org-mode anyway.)  But all I want to do is to publish my notebook either as a blog post of its own, or as part of another block post. 

What's the best way of doing this?  There is a tool called [hugo-jupyter](https://pypi.org/project/hugo_jupyter/), but I can't get it to work.  And in any case, I have no idea if that's the standard method or not.

Many thanks,
Alasdair

**Comment by kaushalmodi** [comment]
Hi Alasdair,

I don't have experience with Jupyter. I'll convert this issue to a discussion so that it stays open. That way it stays more visible for anyone browsing or searching through the discussions.

**Comment by kaushalmodi** [comment]
An online search for "pandoc Jupyter" led me to https://predictablynoisy.com/posts/2019/2019-11-11-ipynb_pandoc/. 

May be you convert Jupyter to Org (and convert to md for Hugo), or directly convert to Md using pandoc (and have Hugo process them as page bundle page resources or static) or convert directly to HTML using pandoc (and have Hugo process them as static).

**Reply by amca01** [reply]
Thank you very much!  That looks just perfect.  It turns out that the version of pandoc on my (Arch Linux) system is a very old one, but pandoc is now in the standard repository and up to date, so that I can install it.  I'll see how I go with it, and report back!
